src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -145,9 +151,31 @@ export default App.extend({
     const filters = this.getState().get('filters');
     const owner = Radio.request('entities', 'clinicians:model', filters.clinicianId);
 
-    this.showChildView('title', new ScheduleTitleView({
+    const scheduleTitleView = this.showChildView('title', new ScheduleTitleView({
       model: owner,
+      showOwnerDroplist: this.canViewAssignedActions,
     }));
+
+    if (this.canViewAssignedActions) {
+      const ownerDroplistView = scheduleTitleView.showChildView('owner', new OwnerDroplist(this.getOwnerFilterOptions(owner)));
+
+      this.listenTo(ownerDroplistView, {
+        'change:owner'({ id }) {
+          this.setState({ filters: { ...filters, clinicianId: id } });
+        },
+      });
+    }
+  },
+  getOwnerFilterOptions(owner) {
+    const options = {
```

Comment:
I wonder if this filter would be better as `align: 'right'`... _probably_?

Reply:
Added the `align: 'right'` in my most recent changes I just pushed.